gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -138,7 +162,7 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
                 credentials=credentials,
                 credentials_file=credentials_file,
                 ssl_credentials=ssl_credentials,
-                scopes=scopes or self.AUTH_SCOPES,
+                scopes=scopes,
```

Comment:
I am assuming this is the same as "pass scopes when self-signed JWT isn't used".

However, I am confused why grpc needs to know scopes when the access token is being handled by the auth library.

Reply:
`grpc_helpers.create_channel` is supposed to somewhat centralize the creation of the `google.auth `credential object (if it is `None`). It looks like this transport *also* initializes a credential if it's `None`, but I think it is redundant and am working on getting it removed in a separate PR.

https://github.com/googleapis/python-api-core/blob/b4860fec797f7d9f02a7241b2b6cef2d27f96f28/google/api_core/grpc_helpers.py#L195